lib/node/runner/checker_ballot_transaction.go

```diff
@@ -213,6 +213,13 @@ func ValidateTx(st *storage.LevelDBBackend, config common.Config, tx transaction
 		return errors.BlockAccountDoesNotExists
 	}
 
+	// check, multiple operation from frozen account.
+	if ba.IsFrozen() {
```

Comment:
Why is this checked ? Was there a bug before ?

Reply:
Originally it was @soonkuk's change. 
It can be removed i think